arkouda-env.yml

```diff
@@ -6,11 +6,26 @@ dependencies:
   - numpy>=1.16.5,<=1.21.5
   - pandas>=1.1.0
   - pyzmq>=20.0.0
+  - typeguard==2.10.0
   - tabulate
   - pyfiglet
   - versioneer
   - matplotlib
   - h5py
   - pip
+  - types-tabulate
+  - tables
+  - pyarrow
+
+  # Developer dependencies
+  - pexpect
+  - pytest
+  - pytest-env
+  - Sphinx
+  - sphinx-argparse
+  - sphinx-autoapi
+  - typed-ast
+

```

Comment:
This file is intended for installs for users, they will not need the developer dependencies. These can be removed.